devops/scripts/benchmarks/html/scripts.js

```diff
@@ -927,11 +962,22 @@ function setupToggles() {
     if (customRangesParam !== null) {
         customRangeToggle.checked = customRangesParam === 'true';
     }
+
+    if (archivedDataToggle && archivedParam !== null) {
+        archivedDataToggle.checked = archivedParam === 'true';
+
+        if (archivedDataToggle.checked) {
+            loadArchivedData();
+        }
+    }
```

Comment:
At some point we might need to add abstraction for "toggles", adding new ones by hand is annoying. But no need to do it now.